dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-create-a-composite-drawing.md

```diff
@@ -13,7 +14,7 @@ This example shows how to use a <xref:System.Windows.Media.DrawingGroup> to crea
 ## Example  
  The following example uses a <xref:System.Windows.Media.DrawingGroup> to create a composite drawing from the <xref:System.Windows.Media.GeometryDrawing> and <xref:System.Windows.Media.ImageDrawing> objects. The following illustration shows the output that this example produces.  
   
- ![A DrawingGroup with multiple drawings](./media/graphicsmm-simple.jpg "graphicsmm_simple")  
+ ![Illustration of a composite drawing showing a square filled with kiwi slices overlapping a black rimmed, green circle on the upper left with a black rimmed, green circle overlapping on the bottom right. There is a gray square border marking the bounds of the drawing.](./media/graphicsmm-simple.jpg "graphicsmm_simple")  
```

Comment:
This is a little long for alt text.  Cut out the parts of this that are in the surrounding text (like the last sentence).